README.md

```diff
@@ -34,12 +34,14 @@ class MainActivity : AppCompatActivity() {
         // ....
 
         // Initialize Volumedeck SDK
+        // Make sure to ask Location permission, and Notification permission (if running on Android 12+)
```

Comment:
```suggestion
        // Make sure to ask for Location and Notification permissions, if running on Android 12+
```

Reply:
If running on android 12+ is for notification only

Or maybe we can remove this android 12+ line
Because no harm in asking notification permission on below 12, because those devices already have this, and developers have to add this anyway because Android 12 is common now

Reply:
Updated my suggestion

Reply:
I think we already added complete segment for permissions, so we can remove this from here
Or we can mention something like
Please make sure to read permission segment 
Or keep permissions part above this initialization part

Reply:
Removed from here and added below